lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "6.2.14",
+  "version": "6.2.15",
```

Comment:
why bump version?

Reply:
I can not publish as 6.2.14 because there is tag v6.2.14 already due to fork of web3-react

Reply:
we can delete the tag